docs/markdown/snippets/find_program-version.md

```diff
@@ -0,0 +1,5 @@
+## found programs now have a version method
+
+The return value of [[find_program]] can now check the exact version of the
+found program, independent of the minimum version requirement. This can be used
+e.g. to perform different actions depending on the exact version detected.
```

Comment:
nitpicking: could tell the new method is `prog.version()`.

Reply:
I mentioned the "version" method in the markdown title, which I'm not sure can (or should) do anything complicated with formatting, and TBH it's exactly comparable to `dependency()`. Too lazy to change it. :D

Reply:
ok fair enough.